go/systests/team_seitan_invite_v2_test.go

```diff
@@ -274,6 +274,8 @@ func TestTeamHandleMultipleSeitans(t *testing.T) {
 	})
 	tt.users = append(tt.users, ann)
 
+	ann.tc.G.Env.Test.SkipSendingSystemChatMessages = true
```

Comment:
so the `suppressTeamChatAnnounce: true` above did not do its thing?

Reply:
ah ok, it's very limited in what it does. maybe it should be factored out and the new `Env.Test` variable should be used

Reply:
ok, let me do that, i didn't even see it

it wasn't working...